cernopendata/modules/fixtures/data/docs/cms-guide-trigger-system/cms-guide-trigger-system.md

```diff
@@ -61,3 +61,26 @@ Find the software and usage instructions in
 
 - [Analysis code for extracting the trigger information from the CMS 2010 data](/record/5003)
 - [Analysis code for extracting the trigger information from the CMS 2011 data](/record/5004)
+
+### HLT Trigger Path definitions
+
+For each dataset, the possible HLT trigger paths are listed in the dataset record. Each HLT path is composed of different algorithms and they are defined in the data-taking configuration files, which are given in 
+
+- [High-Level Trigger information for 2010 CMS open data](/record/FIXME)
+- [High-Level Trigger information for 2011 CMS open data](/record/1700)
+- [High-Level Trigger information for 2012 CMS open data](/record/1701)
+
+The exact parameters (such as pT or eta cuts, or else) for each component of the HLT path can be found in the data-taking configuration file.
+As an example, if you looking for information about `HLT_Dimuon10_Jpsi` trigger, look for a string `process.HLT_Dimuon10_Jpsi`
+in the data-taking configuration file corresponding to the event range of your interest. 
+You will find all components of this HLT path.  You can then find the parameters for each of the components 
```

Comment:
In the TSG world, the "components" are called modules (actually some of them are sequences).  So I would replace "You will find all components of this HLT path" and what follows with something like "You will find the corresponding CMSSW path, which consists of all the modules (i.e., specifically python-configured CMSSW code)  and/or sequences (groups of modules)  that make up that trigger path.  You can then look for each of these modules or sequences in the same configuration file until your find the parameters with which they are configured."

Reply:
Better, indeed, done. In consequence, I've changed "components" to "modules" in the following phrase  as well(for the example case od hltDimuon10JpsiFiltered).